web/src/server_events.js

```diff
@@ -285,8 +279,8 @@ export function force_get_events() {
     get_events_timeout = setTimeout(get_events, 0);
 }
 
-export function home_view_loaded() {
-    waiting_on_homeview_load = false;
+export function finished_initial_fetch() {
+    waiting_on_initial_fetch = false;
```

Comment:
This cluster of renames could be a nice prep commit.

Reply:
Done